### PR TITLE
chore: add some comments

### PR DESCRIPTION
**Pull request by sailorlqh**
I think this would be helpful for users if they set the scope to folder level but didn't grant the service account the asset viewer role to the folder level.

### CODE REVIEW

File: pkg/server/config.go [hunk 1]
```diff
@@ -47,6 +47,8 @@ type HandlerConfig struct {
 type MappingHandlerConfig struct {
 	// DefaultResourceScope is the default resource scope to search resources.
 	// This is only used for global resources such as GCS bucket.
+	// Please make sure your run-pmap service account was granted the permission
```
Comment (capri-xiyue) [inline pkg/server/config.go:50]: Nits: Please make sure the Service Account used in the Cloud Run service for Data Mapping is granted the `roles/cloudasset.viewer` to the corresponding scope level.

 run-pmap service account is confusing for people who are not familiar with the implementations details of  our terraform module

Comment (sailorlqh) [reply pkg/server/config.go]: Thanks! Updated.

File: pkg/server/config.go [hunk 2]
```diff
@@ -47,6 +47,9 @@ type HandlerConfig struct {
 type MappingHandlerConfig struct {
 	// DefaultResourceScope is the default resource scope to search resources.
 	// This is only used for global resources such as GCS bucket.
+	// Please make sure the Service Account used in the Cloud Run service for
+	// Data Mapping is granted the roles/cloudasset.viewer to the corresponding
```
Comment (capri-xiyue) [inline pkg/server/config.go:51]: make `roles/cloudasset.viewer` clear. Probably a single quote

Comment (sailorlqh) [reply pkg/server/config.go]: Done

**Review by capri-xiyue — APPROVED** [review]

**Pull request merged**